### PR TITLE
{bio}[foss/2018b] Bowtie v1.2.2

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by smoors** [comment]
@boegel I think `Bowtie2-2.3.4.2-foss-2018b.eb` is the problem here: the toolchain for `tbb` should not be specified?

**Comment by boegel** [comment]
@smoors Indeed, please remove the hardcoded subtoolchain in `Bowtie2-2.3.4.2-foss-2018b.eb`, it shouldn't be there.

File: easybuild/easyconfigs/b/Bowtie/Bowtie-1.2.2-foss-2018b.eb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+description = """Bowtie is an ultrafast, memory-efficient short read aligner.
+ It aligns short DNA sequences (reads) to the human genome."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+toolchainopts = {'pic': True, 'cstd': 'gnu++98'}
+
+sources = ['%(namelower)s-%(version)s-src.zip']
+source_urls = ['http://download.sourceforge.net/bowtie-bio/']
+checksums = [
+    '806d618d073107e75dec6c3a61fc9ce4a1eb91657e75cb1bfa9ca2bf926482ca',  # bowtie-1.2.2-src.zip
```
Comment (boegel) [inline easybuild/easyconfigs/b/Bowtie/Bowtie-1.2.2-foss-2018b.eb]: @smoors Please make this a single line (since there's only one source & no patches)

**Comment by boegel** [comment]
@smoors #6926 is merged, so let's deal with the tine style issue to make this PR good to go?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3101.skitty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/3557ee1b766a5094af7ad17eec559681 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2409.golett.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/355df92956179cb41ecc171cbbc26941 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2086.delcatty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/4a9a61ae058c7c723a68af36e37658db for a full test report.

**Comment by boegel** [comment]
Going in, thanks @smoors!

**Pull request merged**